Gulpfile.js

```diff
@@ -53,6 +60,97 @@ function rename(fn) {
   });
 }
 
+/**
+ *
+ * @typedef {("asserts" | "builders" | "constants" | | "validators")} HelperKind
+ * @param {HelperKind} helperKind
+ */
+function generateTypeHelpers(helperKind) {
+  const dest = `./packages/babel-types/src/${helperKind}/generated/`;
+  const formatCode = require("./scripts/utils/formatCode");
+  return gulp
+    .src(".", { base: __dirname })
+    .pipe(errorsLogger())
+    .pipe(
+      through.obj(function (file, enc, callback) {
+        file.path = "index.js";
+        file.contents = Buffer.from(
+          formatCode(
+            require(`./packages/babel-types/scripts/generators/${helperKind}`)(),
+            dest + file.path
+          )
+        );
+        fancyLog(`${chalk.green("✔")} Generated ${helperKind}`);
+        callback(null, file);
+      })
+    )
+    .pipe(gulp.dest(dest));
+}
+
+function buildFlowTypings() {
+  const dest = "./packages/babel-types/lib/";
+  return gulp
+    .src(".", { base: __dirname })
+    .pipe(errorsLogger())
+    .pipe(
+      through.obj(require("./packages/babel-types/scripts/generators/flow.js"))
+    )
+    .pipe(rename(() => "index.js.flow"))
+    .pipe(gulp.dest(dest));
+}
+
+function buildTypeScriptTypings() {
```

Comment:
Nit: since those two functions are almost the same, we could use
```js
function buildTypings(generator: "flow" | "typescript", output: "index.js.flow" | "index.d.ts");
```